ocpmodels/trainers/energy_trainer.py

```diff
@@ -209,7 +209,9 @@ def predict(self, loader, results_file=None, disable_tqdm=False):
 
     def train(self):
         self.best_val_mae = 1e9
-        for epoch in range(self.config["optim"]["max_epochs"]):
+        
+        start_epoch = self.start_step // len(self.train_loader)
+        for epoch in range(start_epoch, self.config["optim"]["max_epochs"]):
```

Comment:
Instead of rounding down to resume from the beginning of the epoch, would it be better to resume from `self.start_step` directly and run till `self.config["optim"]["max_epochs"] * len(self.train_loader)`?

Reply:
@anuroopsriram did you have thoughts on this?